offchain-modules/lib/src/dapp/relayer/ckb_relayer.rs

```diff
@@ -1 +1,125 @@
+use crate::transfer::to_eth::unlock;
+use crate::util::config::ForceConfig;
+use crate::util::eth_util::{parse_private_key, Web3Client};
+use anyhow::{anyhow, bail, Result};
+use ethereum_types::H256;
+use futures::future::join_all;
+use log::{error, info};
+use serde::{Deserialize, Serialize};
+use shellexpand::tilde;
+use sqlx::MySqlPool;
+use std::ops::Add;
+use std::time::Instant;
+use tokio::time::Duration;
 
+#[derive(Clone, Default, Debug, Serialize, Deserialize, sqlx::FromRow)]
+pub struct CkbToEthRecord {
+    pub id: u32,
+    pub ckb_burn_tx_hash: String,
+    // pub status: String,
+    // pub recipient_addr: Option<String>,
+    // pub token_addr: Option<String>,
+    // pub token_amount: Option<String>,
+    // pub fee: Option<String>,
+    // pub eth_tx_hash: Option<String>,
+    // pub err_msg: Option<String>,
+    pub ckb_spv_proof: Vec<u8>,
+    pub ckb_raw_tx: Vec<u8>,
+}
```

Comment:
Please remember to import `CkbToEthRecord` from db/indexer.rs after indexer-pr merged.